spec/Helpers.coffee

```diff
@@ -1517,22 +1635,23 @@ describe 'Component traits', ->
                 y.disconnect()
             , delay*req.num
 
-      noflo.helpers.WirePattern c,
-        in: ['x', 'y']
-        out: 'out'
-        async: true
-        forwardGroups: true
-        group: isUuid
-        gcFrequency: 2 # every 2 requests
-        gcTimeout: 0.02 # older than 20ms
-      , (input, groups, out, done) ->
-        setTimeout ->
-          out.send
-            id: groups[0]
-            x: input.x
-            y: input.y
-          done()
-        , 3
+      before ->
+        noflo.helpers.WirePattern c,
+          in: ['x', 'y']
+          out: 'out'
+          async: true
+          forwardGroups: true
+          group: isUuid
+          gcFrequency: 2 # every 2 requests
```

Comment:
`gc` options only work in legacy mode i suppose, thus should be removed from here. Also, is `group` by `RegExp` supported?